app/controllers/metrics_controller.rb

```diff
@@ -4,7 +4,13 @@ def index
 		course = Course.find_by(name: params[:course_name])
 		@course_grace_days = course.grace_days
 		@num_assessments = course.assessments.count()
-		@max_consecutive_assessments = course.assessments.group("category_name").count().max()[1] - 1
+		
+		course_max = course.assessments.group("category_name").count().max()
+		if (course_max == nil)
+			@max_consecutive_assessments = 0
+		else
+			@max_consecutive_assessments = course.assessments.group("category_name").count().max()[1] - 1
```

Comment:
nit: could we change `course.assessments.group("category_name").count().max()` to just `course_max`?

Reply:
good catch! done